src/utils.ts

```diff
@@ -1328,21 +1328,8 @@ export function shuffle<T>(sequence: Iterable<T>, limit = 0): Array<T> {
 
 // TODO: this should be codified in command construction
 // @see https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#read-concern
-export function commandSupportsReadConcern(command: Document, options?: Document): boolean {
-  if (command.aggregate || command.count || command.distinct || command.find || command.geoNear) {
-    return true;
-  }
-
-  if (
-    command.mapReduce &&
-    options &&
-    options.out &&
-    (options.out.inline === 1 || options.out === 'inline')
-  ) {
-    return true;
-  }
```

Comment:
Is it okay to remove this here? If I follow the code correctly, this method is called for all commands, including `client.db().command()`, so this change would also affect users who run `mapReduce` through that instead of the helper (and since the migration notes mention this as an alternative for cases in which users can’t use agg pipelines, that seems to be a supported use case).

Reply:
I think this is a judgement call.  Do you have an opinion? 

When a user manually constructs the command, they can attach their own read concern.  I think it's reasonable to say that because we're dropping support for `mapReduce`, that means we drop support for attaching `readConcern`s automatically when executing a runCommand.

Alternatively, we could continue to support automatically attaching the read concern to `mapReduce` commands.  One argument in favor of this approach is that the `Db.command()` options does take a read concern, so users may assume that the read concern they supply as a part of the options object would get attached to their command.

Reply:
> the `Db.command()` options does take a read concern, so users may assume that the read concern they supply as a part of the options object would get attached to their command.

I don’t have a strong opinion, but this seems like a reasonable expectation to me, esp. if this works for other commands.

Reply:
I think it's reasonable to to keep it in until the driver's minimum server version has completely removed the command.

Reply:
Just a note as well because it seemed a bit unclear here. This is only determining if the command supports read concern - it still must be explicitly provided. This change would have just ignored the provided one in the inlined `mapReduce` case.